services/movie.py

```diff
@@ -0,0 +1,39 @@
+from db.models import Movie
+
+
+def get_movies(genres_ids: list[int] = None, actors_ids: list[int] = None):
+    if genres_ids is None and actors_ids is None:
+        return Movie.objects.all()
+    elif genres_ids is not None and actors_ids is not None:
+        return Movie.objects.filter(
+            genres__id__in=genres_ids,
+            actors__id__in=actors_ids
+        )
+    elif genres_ids is not None and actors_ids is None:
+        return Movie.objects.filter(
+            genres__id__in=genres_ids
+        )
+    elif genres_ids is None and actors_ids is not None:
+        return Movie.objects.filter(
+            actors__id__in=actors_ids
+        )

```

Comment:
It is a bad approach. Kate is right. Get all movies and then filter it for every argument passed.